CONTRIBUTING.md

```diff
@@ -2,6 +2,10 @@
 
 ## Getting Started
 
+### Code of Conduct
+
+On <https://github.com/parthatom/Grid/CODE_OF_CONDUCT.md> you can find the code of conduct for an efficient working of this community.
```

Comment:
wording: You can find our code of conduct at <link>. We ask that all members of the community follow this code in order to promote a welcoming and productive environment.